src/main/java/seedu/address/logic/parser/AddCommandParser.java

```diff
@@ -45,10 +48,32 @@ public AddCommand parse(String args) throws ParseException {
                         PREFIX_LICENCE_PLATE, PREFIX_ADDRESS, PREFIX_TAG, PREFIX_POLICY_NUMBER,
                         PREFIX_POLICY_ISSUE_DATE, PREFIX_POLICY_EXPIRY_DATE);
 
+        if (!argMultimap.getPreamble().isEmpty()) {
+            throw new ParseException(String.format(MESSAGE_INVALID_COMMAND_FORMAT, MESSAGE_PREAMBLE_DETECTED));
+        }
+
         if (!arePrefixesPresent(argMultimap, PREFIX_NAME, PREFIX_PHONE, PREFIX_EMAIL, PREFIX_NRIC,
-                PREFIX_LICENCE_PLATE, PREFIX_ADDRESS)
-                || !argMultimap.getPreamble().isEmpty()) {
-            throw new ParseException(String.format(MESSAGE_INVALID_COMMAND_FORMAT, AddCommand.MESSAGE_USAGE));
+                PREFIX_LICENCE_PLATE, PREFIX_ADDRESS)) {
+            String errorMessage = MESSAGE_MISSING_FIELDS_FOR_ADD_COMMAND;
+            if (argMultimap.getValue(PREFIX_NAME).isEmpty()) {
+                errorMessage += "- Name(n/) ";
```

Comment:
I think a good practice here should be using PREFIX_NAME instead of "n/" but current version won't cause any error. 
```suggestion
                errorMessage += "- Name("  + PREFIX_NAME + ") ";
```

Reply:
Yup agree with Chai Yin, since we already have the prefix declared we should use it rather than hard coding the string itself